### ISSUE TITLE
tell user about undetected spikes

### ISSUE THREAD

**Issue by nsteinme** [MouseLand/Kilosort2#3]
One idea: run a naive threshold-crossing detection at the end and report times, channels, and amplitudes of anything that is left? have text or graphical output of the rate of such spikes relative to the rate of spikes that were detected?

**Comment by marius10p** [comment]
A lot more stuff gets detected now with the threshold in the final pass being so low (ops.Th(2) = 4). There should be very little left, but let's wait for user reports.

**Comment by czuba** [comment]
Just bumping this issue.

Some kind of **post-sort thresholded output** would be particularly useful for:
• **catching undetected isolatable spikes** (informing user parameterization issues, and possibly codebase issues)
• allowing some level of user-interpretable **evidence of silently/automatically dropped clusters** that the user may want/need to recover (...also useful for refining sort & merge parameters)
• providing an **equivalence to classical multi-unit responses** (e.g. hand-sorted outputs that are clearly neither noise, nor purely isolatable SU)

**Comment by susuchen66** [comment]
Hi we see missing spikes very often while sorting different dataset, including long session or short epochs. Lowers two threshold, minimal firing rate per channel, etc didn't help. Missing spikes are observable either in the Matlab GUI Nick developed or Phy. We used JRClust to sort same dataset and compared the result with KS2. We found large amplitude spikes dropped out over large stretches of session in KS2 but not in JRClust. 

![missingspk7](https://user-images.githubusercontent.com/29060635/60773928-a00c8300-a0da-11e9-9ee9-b013d2a007d2.PNG)
![missingspk4](https://user-images.githubusercontent.com/29060635/60773932-af8bcc00-a0da-11e9-8470-7c393bcb032a.PNG)
![missingspk_Phy](https://user-images.githubusercontent.com/29060635/60773935-b286bc80-a0da-11e9-8e87-1c8886ea1c49.PNG)

Comparison between JRClust output (left) and KS2 output (right) on the same unit:
![image-2](https://user-images.githubusercontent.com/29060635/60773952-e2ce5b00-a0da-11e9-86f3-26d35583ba04.png)
